src/compiler/sys.ts

```diff
@@ -340,11 +340,20 @@ namespace ts {
                     }
 
                     function fileChanged(curr: any, prev: any) {

```

Comment:
After getting offline help, I don't think this is worth fixing as part of this change.